pleskdistup/main.py

```diff
@@ -615,6 +615,12 @@ def main():
                 log.debug(f"Removed the resume file {options.resume_path!r}")
             os.unlink(options.completion_flag_path)
 
+    # After lock there's no need to manage signals in such a detailed manner
+    # Simply log the reason for not rebooting
+    exit_signal_handler = create_exit_signal_handler(util_name)
+    for signum in (signal.SIGINT, signal.SIGTERM, signal.SIGHUP, signal.SIGABRT):
+        signal.signal(signum, lambda signum, frame: log.info(f"Received signal {signum}, going to exit..."))
```

Comment:
maybe you can unify both `for signum..` constructs under one `def assign_killing_signals..(ff: Callable)`